### PR TITLE
Add currently hovered tile to TileIndicators

### PR DESCRIPTION
**Pull request by deathbeam**
Add tile that is currently being hovered to tile indicators.

Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

Preview:
peek 2018-06-05 17-11

### CODE REVIEW

File: runescape-api/src/main/java/net/runelite/rs/api/RSClient.java [hunk 1]
```diff
@@ -127,6 +127,16 @@
 	@Import("gameState")
 	int getRSGameState();
 
+	@Import("checkClick")
+	@Override
+	void setCheckClick(boolean checkClick);
+
+	@Import("mouseX2")
+	void setMouseX2(int mouseX2);
```
Comment (Adam-) [inline runescape-api/src/main/java/net/runelite/rs/api/RSClient.java]: Maybe these should be given better names now?

Comment (deathbeam) [reply runescape-api/src/main/java/net/runelite/rs/api/RSClient.java]: Yeah, i mostly just wanted to avoid PR to internal due to lazyness :d

Comment (deathbeam) [reply runescape-api/src/main/java/net/runelite/rs/api/RSClient.java]: Updated @Adam-

**Comment by deathbeam** [comment]
Any update on this @Adam- ?

**Pull request merged**